docs/index.md

```diff
@@ -1,13 +1,19 @@
 # Introduction
 
-This is the documentation page of [Skipper](https://github.com/zalando/skipper). Skipper is an HTTP router and reverse proxy for service composition. It's designed to handle >100k HTTP route definitions with detailed lookup conditions, and flexible augmentation of the request flow with filters. It can be used out of the box or extended with custom lookup, filter logic and configuration sources.
+This is the documentation page of
+[Skipper](https://github.com/zalando/skipper). Skipper is an HTTP
+router and reverse proxy for service composition. It's designed to
+handle large amounts of HTTP route definitions (>800000 routes) with
```

Comment:
what do you think about adding here something like "dynamically configured routes", or something to emphasized that there's no need to restart on config?